test/studies/bigint/bigintops.graph

```diff
@@ -0,0 +1,11 @@
+perfkeys: Elapsed 256 assign time =, Elapsed 1024 assign time =, Elapsed 256 on stmt time =
+files: bigintops.dat, bigintops.dat, bigintops.dat, bigintops.dat, bigintops.dat
+graphkeys: 256 bit addition, 1024 bit addition, 256 bit assign, 1024 bit assign, 256 bit pow
```

Comment:
There's 3 perfkeys, but 5 files and graphkeys here. Does this work?

Reply:
Hmm, apparently yes, it does work, but, that was unintentional and is news to me... fixed.